src/configimpl.l

```diff
@@ -1713,6 +1713,30 @@ void Config::checkAndCorrect()
     dotPath="";
   }
 
+  // check dot path
+  QCString &dotCachedir = Config_getString(DOT_CACHEDIR);
+  if (!dotCachedir.isEmpty())
+  {
+    QFileInfo fi(dotCachedir);
+    if (fi.exists() && fi.isDir())
+    {
+      dotCachedir += '/';
+    }
+    else
+    {
+        warn_uncond("The dot cache folder (DOT_CACHEDIR) %s could not be found (ignoring).\n",dotCachedir.data());
+        dotCachedir="";
+    }
+#if defined(_WIN32) // convert slashes
+    auto l=dotCachedir.length();
+    for (int i=0;i<l;i++) if (dotCachedir.at(i)=='/') dotCachedir.at(i)='\\';
+#endif
```

Comment:
This should be handled with `substitute` and `Portable::pathSeparator`

Reply:
Ok, this is C&P code from a few lines above. Should those be changed, too (and be put in a separate PR?)

Reply:
Amazing that this is in (in my opinion) and the amount of `_WIN32` / `WIN32` still present outside portable.cpp.
In doxygen.cpp I found also `static QCString fixSlashes(QCString &s)` I think a candidate to move to portable.cpp and to be used.

Reply:
@albert-github: Ok, so should I try to migrate the code or are you willing to do so (as you are probably faster. ;) )?

Reply:
Thanks for the confidence. I'll leave this patch to you.

Reply:
Not what I was hoping for. I'll see when I'll get around to it. Worst case I'd leave it like it is for now....

Reply:
As this is C&P code and this PR is not about fixing that code, I'd keep it the way it is and maybe fix it later (but to be honest; I have other points on my agenda that I'm more interested in improving).